src/LuaSystemBody.cpp

```diff
@@ -276,6 +276,28 @@ static int l_sbody_attr_apoapsis(lua_State *l)
 	return 1;
 }
 
+
+/*
+ * Attribute: orbitPeriod
+ *
+ * The orbit of the body, around its parent, in days
```

Comment:
Fixed. Said float now. (although what "days in float" mean might be confuding, due to babylonian base 60 system)